proto/serverless/proto/instrumentation/tagset/v1/tagset.proto

```diff
@@ -0,0 +1,272 @@
+syntax = "proto3";
+
+package serverless.proto.instrumentation.tagset.v1;
+
+message Tags {
+  // The required set of tags for The Serverless Ingest Platform. Payloads without all these tags will be rejected.
+  SlsSdkTags sls = 1;
+
+  // ============================================ //
+  // Defined TagSets start at field number 4000  //
+  // ========================================== //
+
+  // The root AWS Lambda Span tags.
+  optional AwsLambdaTags aws_lambda_tags = 4000;
+
+  // The AWS Lambda Initialization tags.
+  optional AwsLambdaInitializationTags aws_lambda_initialization_tags = 4001;
+
+  // The AWS Lambda Invocation tags.
+  optional AwsLambdaInvocationTags aws_lambda_invocation_tags = 4002;
+
+  // The AWS SDK Tags.
+  optional AwsSdkDynamodbTags aws_sdk_dynamodb_tags = 4003;
+  optional AwsSdkSqsTags aws_sdk_sqs_tags = 4004;
+  optional AwsSdkSnsTags aws_sdk_sns_tags = 4005;
```

Comment:
Do we envision normalizing tags into flat structure on JS side so we end with names as `aws.lambda.arch`, `aws.lambda.invocation.duration` etc.?

I'm rising this, as at https://github.com/serverlessinc/platform-core/pull/751#discussion_r932113794 I've proposed some generic approach that, if followed, allows us to end with such a flat structure easily, and I'm not sure if the same can be achieved with names as proposed here.

Or do you propose here, that final tag names would be as: `aws_lambda_tags.arg`, `aws_lambda_invocation_tags.duration` etc. ?

Reply:
Probably from a display sense we will want to do flattened tags for the UI. On ingest though I don't really see the reason to do it. If we create the expectation that we either flatten from all the keys to the value or replace `_` with a `.` I think that should be fine.

Reply:
>  If we create the expectation that we either flatten from all the keys to the value or replace _ with a . I think that should be fine.

If we decide to translate `_`  to `.`, then you remove the possibility of using `_` in end tag names.
e.g.  instead of `aws.api_gateway.request.query_string_parameters` we'll get `aws.api.gateway.request.query.string.parameters`, which doesn't feel good.

So what you propose here doesn't seem to allow nice translation to the UI, while that's taken care of cleanly in the approach I've proposed.

Reply:
Okay, that's a good point. Let me think on the approach you suggested. I'm worried that will add a lot of nesting within the schema that I'd like to avoid if we can.

Reply:
> I'm worried that will add a lot of nesting within the schema that I'd like to avoid if we can.

It'll increase nesting, but I don't think it'll end as hardly maintainable. We should be good, as long as it'll be well ordered in the file.